BP/common/culture/cultures/00_somalian.txt

```diff
@@ -0,0 +1,348 @@
+﻿somalian_group = {
```

Comment:
In CK2 this was the Cushitic group in the mod, shouldn't it be named that here too?

Reply:
somalian_group is the vanilla code name, the actual name is Horn Africans. Cushitic doesn't work really with the expansion south

Reply:
Unless I'm completely off, don't the Cushitic languages and cultures cover the horn of Africa?

![image](https://user-images.githubusercontent.com/7620180/103553916-6d4d7180-4eae-11eb-89bf-364e9d6a67d9.png)

Reply:
Yeah, actually I was mixing up this with the East African group. They both share Cushitic languages which make up culture groups

Reply:
What culture designating a Cushitic language does the East African group include? Nubian, Welayta, Coptic and Daju aren't associated with Cushitic?

Reply:
I took that Welayta represents the much fought over Omotic languages and also kinda the Oromo considering what the religion is called?

Reply:
But Wolaytta and Omotic aren't tied to Cushitic languages, from what I see? It is only Oromo that is Cushitic, which isn't tied to Omotic.

Reply:
From their Wikipedia page there's a scientific argument that they are but what I was going for was that their religion was called the Oromo-Somali by PDX in vanilla so I thought that's what they represented. At least that's what I though anyway.

Reply:
I believe the religion isn't a relevant indicator when it comes to culture groups, with that in mind I personally think the Cushitic name can be used instead of the generic Horn Africans. The argument between linguists about Omotic languages maybe being a "West Cushitic" branch doesn't change the fact that they would still be distinct from all the "East Cushitic" languages, therefore i believe very few players would have that reference, especially since East African includes other cultures as well.

Reply:
Okay then.

Reply:
Renamed